examples/sample-app/README.md

```diff
@@ -75,13 +83,35 @@ All commands assume the `openshift` binary is in your path (normally located und
 
     Note: sudo is required so the kubernetenes proxy can manipulate iptables rules to expose service ports.
 
-3. Deploy the private docker registry within OpenShift:
+3. Set up your client to reach the OpenShift master now running.
 
-        $ openshift cli apply -f docker-registry-config.json
+    Since OpenShift services are secured by TLS, your client will
+    need to accept the server certificates and present its own client
+    certificate. These are generated as part of the `openshift start`
+    command in whatever the current directory is at the time. You will
+    need to point osc and curl at the appropriate CA bundle and client
+    key/cert in order to connect to OpenShift. Assuming you are running as
+    a user other than root, you will also need to make the private client
+    key readable by that user. (Note: this is just for example purposes;
+    in a real installation, users would generate their own keys and not
+    have access to the system keys.)
 
-4. Confirm the registry is started (this can take a few mins):
+        $ export KUBECONFIG=openshift.local.certificates/admin/.kubeconfig
+        $ export CURL_CA_BUNDLE=openshift.local.certificates/admin/root.crt
+        $ sudo chmod +r openshift.local.certificates/admin/key.key
 
-        $ openshift cli get pods
+4. Deploy a private docker registry within OpenShift using the sample definition in this directory:
```

Comment:
That script isn't about running the docker registry securely (though admittedly that'd be nice too), it's about setting it up to talk securely to the openshift server over https when making imagerepository calls.

so the docker daemon config is still needed, unfortunately, I think.